blocks/how-to/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@blocks/how-to",
+  "version": "0.1.0",
+  "description": "Describe a task or process via a list of steps ",
+  "keywords": [
+    "blockprotocol",
+    "blocks"
+  ],
+  "license": "MIT",
+  "author": "luisbettencourt",
+  "scripts": {
+    "build": "block-scripts build",
+    "dev": "block-scripts dev",
+    "format": "prettier --write --ignore-unknown .",
+    "generate-types": "block-scripts codegen",
```

Comment:
It's `schema` in the block templates – I'm not sure what the best name is, welcome suggestions.

Maybe we should just make it `schema` everywhere for now since that's what other blocks have.

Reply:
I think the two sensible routes are:

- Stick with the template for now (`schema`)
- Change the template to standardise on something else (I would suggest `codegen`, which could be split into multiple steps later like `codegen:block-entity-type` or something)

Reply:
Cannot be codegen until we make that work in the turbo pipeline

Reply:
Unless we suffix it so it isn't picked up by the pipeline

Reply:
Forgot turbo-repo isn't that configurable :( Annoying that the monorepo is polluting scripts in templates we're publishing

Reply:
Changed script name to schema in bd02c4ddc0f98103d1ce98672df36f13028b7140